src/SearchInput/SearchInput.tsx

```diff
@@ -11,19 +12,29 @@ export type SearchInputItem = {
 
 interface IContainer {
   show: boolean
+  hasBorder?: boolean
+}
```

Comment:
you can just extend `IInnerContainer` interface instead of adding the props to all the others so it looks more clean